kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala

```diff
@@ -183,19 +183,47 @@ private[v1] class OperationsResource extends ApiRequestContext with Logging {
             i.getSetField.name(),
             i.getSetField match {
               case TColumnValue._Fields.STRING_VAL =>
-                i.getStringVal.getFieldValue(TStringValue._Fields.VALUE)
+                if (i.getStringVal.isSetValue) {
```

Comment:
A possible risk is that `getStringVal` on the unset field will throw RuntimeException here.
Maybe use `i.isSetStringVal` for the case condition is better? And make null as default return.

Reply:
I tried to use that, but it does not work. you can have a try.

Reply:
With 
```
case TColumnValue._Fields.STRING_VAL =>
                if (i.isSetStringVal) {
                  i.getStringVal.getFieldValue(TStringValue._Fields.VALUE)
                } else {
                  null
                }
              case TColumnValue._Fields.BOOL_VAL =>
                if (i.isSetBoolVal) {
                  i.getBoolVal.getFieldValue(TBoolValue._Fields.VALUE)
                } else {
                  null
                }
              case TColumnValue._Fields.BYTE_VAL =>
                if (i.isSetByteVal) {
                  i.getByteVal.getFieldValue(TByteValue._Fields.VALUE)
                } else {
                  null
                }
              case TColumnValue._Fields.DOUBLE_VAL =>
                if (i.isSetDoubleVal) {
                  i.getDoubleVal.getFieldValue(TDoubleValue._Fields.VALUE)
                } else {
                  null
                }
              case TColumnValue._Fields.I16_VAL =>
                if (i.isSetI16Val) {
                  i.getI16Val.getFieldValue(TI16Value._Fields.VALUE)
                } else {
                  null
                }
              case TColumnValue._Fields.I32_VAL =>
                if (i.isSetI32Val) {
                  i.getI32Val.getFieldValue(TI32Value._Fields.VALUE)
                } else {
                  null
                }
              case TColumnValue._Fields.I64_VAL =>
                if (i.isSetI64Val) {
                  i.getI64Val.getFieldValue(TI64Value._Fields.VALUE)
                } else {
                  null
                }
```

The result is:

```
ArrayBuffer(null, false, 0, 0.0, 0, 0, 0)
```

Reply:
so, to keep the style align.

I decided to use `i.get*Val.isSetValue` finally.

Reply:
Yeah, I have tried but failed before raising the original comment. Let's keep it as it is.